src/Backend/Compiler.ts

```diff
@@ -58,6 +58,12 @@ interface Compiler {
    */
   getToolchains(toolchainType: string, start: number, count: number): Toolchains;
 
+  /**
+   * @brief Function to return the list of already-installed toolchain
+   * @param toolchainType One of value returned from toolchainTypes()
```

Comment:
```suggestion
   * @brief Function to return the list of already-installed toolchains
   * @param toolchainType One of the values returned from toolchainTypes()
```

(optional, trivial article errors) I don't know if this kinda grammar check is helpful in terms of review or but as I found it out..